src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/FlexTicketDiscountInfo.tsx

```diff
@@ -65,21 +65,19 @@ export const FlexTicketDiscountInfo = ({userProfiles, style}: Props) => {
             const discountPercent = ladder?.steps[ladder.current].discount;
 
             const userProfileName = getReferenceDataName(u, language);
-            const discountText =
-              (discountPercent !== undefined) &&
-              t(
-                PurchaseOverviewTexts.flexDiscount.discountPercentage(
-                  discountPercent.toFixed(0),
-                ),
-              );
+            const discountLabel =
```

Comment:
@rosvik should we keep this here? since the discount ladder shouldn't be relevant if we only shows the new price.

Reply:
Yes, we still want to show some indication that this ticket is or will be discounted.